### PR TITLE
Feature/reworked Edit with angularjs(comments)

### PR DESCRIPTION
**Pull request by OKrain**
[Link to JIRA ticket](https://ssu-jira.softserveinc.com/browse/LR-181)

### Author.

- [x] I have added link to this PR to the JIRA
- [x] I have logged time in JIRA.

### Reviewers.

- [x] @brouhaha88 
- [x] @sergiy17 
- [x] @vovka

### Issue summary.

*Rework edit with AngularJs*

### Fix summary. 

_Reworked edit with AngularJs_

### Testing approach.

There is unit test for editQuiz controller

### CODE REVIEW

**Comment by sergiy17** [comment]
:+1:

File: spec/javascripts/angularjs_app/quizzes_controller_spec.js [hunk 1]
```diff
@@ -101,19 +101,16 @@ describe('editQuizCtrl', function() {
   var $controller, $scope, $routeParams, Quiz;
```
Comment (vovka) [inline spec/javascripts/angularjs_app/quizzes_controller_spec.js]: Remove `$routeParams` here as well

Comment (OKrain) [inline spec/javascripts/angularjs_app/quizzes_controller_spec.js]: Done.

**Comment by vovka** [comment]
lgtm

**Pull request merged**